test-network-function/networking/suite.go

```diff
@@ -78,8 +78,8 @@ var _ = ginkgo.Describe(common.NetworkingTestKey, func() {
 			}
 		})
 
-		for _, containerUnderTest := range containersUnderTest {
-			testNodePort(containerUnderTest.Oc.GetPodNamespace())
+		for _, podUnderTest := range config.PodsUnderTest {
+			testNodePort(podUnderTest.Namespace)
```

Comment:
i guess we need a namespace list from the configInstance as well ...

Reply:
jira created for this